torch/nn/modules/rnn.py

```diff
@@ -1283,6 +1283,10 @@ def __init__(self, input_size: int, hidden_size: int, bias: bool = True,
     def forward(self, input: Tensor, hx: Optional[Tuple[Tensor, Tensor]] = None) -> Tuple[Tensor, Tensor]:
         if input.dim() not in (1, 2):
             raise ValueError(f"LSTMCell: Expected input to be 1D or 2D, got {input.dim()}D instead")
+        if hx is not None:
+            for idx, value in enumerate(hx):
+                if value.dim() not in (1, 2):
+                    raise ValueError(f"LSTMCell: Expected hidden{idx} to be 1D or 2D, got {value.dim()}D instead")
```

Comment:
Hm for consistency with the documentation seems like there will be 2 Tensors in hx here and we call them `h_0` and `c_0` so would prefer not to call this `hidden{idx}`

Alternatively could we could we rephrase this to
```suggestion
                    raise ValueError(f"LSTMCell: Expected hx[{idx}] to be 1D or 2D, got {value.dim()}D instead")
```

Reply:
Thanks for your comment, fixed.